taskchampion/taskchampion/src/server/cloud/gcp.rs

```diff
@@ -242,12 +251,18 @@ mod tests {
     /// indicate anything else.
     fn make_service() -> Option<(GcpService, impl Fn(&str) -> Vec<u8>)> {
         let Ok(bucket) = std::env::var("GCP_TEST_BUCKET") else {
-            return None;
-        };
+            return None;};
+        
+        let Ok(credentialpath) = std::env::var("GCP_CREDENTIAL_PATH") else {
+            return None;};
+            
```

Comment:
Don't forget to run `cargo fmt` on your changes!

Reply:
Okay!